File: internal/symbols/testdata/python/sample.py
def greet(name: str) -> str:
    """Return a greeting message."""
    return f"Hello, {name}!"


def add(a: int, b: int) -> int:
    """Add two integers."""
    return a + b


class Calculator:
    def __init__(self, value: int = 0):
        self.value = value

    def multiply(self, x: int) -> int:
        self.value *= x
        return self.value

File: internal/symbols/testdata/go/sample.go
package sample

import "fmt"

// Greet returns a greeting message.
func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

// Add adds two integers.
func Add(a, b int) int {
	return a + b
}

type Calculator struct {
	Value int
}

func (c *Calculator) Multiply(x int) int {
	c.Value *= x
	return c.Value
}
